components/02-elements/breadcrumbs/_breadcrumbs.scss

```diff
@@ -1,33 +1,56 @@
-$breadcrumbs-padding        : 10px;
-$breadcrumbs-margin         : 10px;
-$breadcrumbs-color          : $font-color-base; 
-$breadcrumbs-color-second   : $gray;
-$breadcrumbs-clickable-area : 48px;
+$breadcrumbs-margin         : 8px !default;
+$breadcrumbs-padding        : 8px !default;
+$breadcrumbs-clickable-area : 48px !default;
+$breadcrumbs-color          : $gray-dark !default;
+$breadcrumbs-font-family    : $font-family-base !default;
+$breadcrumbs-font-size      : $font-size-small !default;
+$breadcrumbs-line-height    : $font-line-height !default;
+//Active
```

Comment:
1. IMO comment is not necessary, we now that is active, it's from code.
2. use BEM notification for variables `$breadcrumbs__padding` etc

Reply:
Done